src/server/platform.ts

```diff
@@ -23,9 +23,11 @@ export function setServerPlatform(document: any, opts: DocumentOptions) {
       if (opts.outDir == null) {
         throw new Error(`Server platform missing "outDir"`);
       }
-      const pathname = url.pathname + '.js';
+      const pathname = !['.js', '.cjs', '.mjs'].includes(extname(url.pathname))
+        ? url.pathname + '.js'
+        : url.pathname;
       const filePath = join(opts.outDir, pathname);
-      return filePath;
+      return './' + filePath;
```

Comment:
What is the reasoning behind adding `./`? It is breaking my code. Not saying it is wrong just trying to understand?